### DISCUSSION TITLE
Framework Publish is throwing timeout exception

### DISCUSSION THREAD

**Discussion by amitpriyadarshi** [Sunbird-Knowlg/Community#183] [Bugs]
@pallakartheekreddy while publishing one of the framwork (`tn_k-12_5`), it throws timeout exception. It is working properly for other framework.
While analysing we found that it is very heavy response and while fetching complete hierarchy from neo4j, it is taking too much time.

**Comment by pallakartheekreddy** [comment]
@amitpriyadarshi Thanks for bringing this issue. We will check.
